### PR TITLE
Device: added getSettingInfo() overloads for getting specific setting info

### PR DESCRIPTION
**Pull request by ncorgan**

### CODE REVIEW

**Comment by ncorgan** [comment]
I wasn't sure what values to bump for the API and ABI versions, so make sure to take a look there.

**Comment by ncorgan** [comment]
Bump

File: include/SoapySDR/Device.h [hunk 1]
```diff
@@ -1346,6 +1346,14 @@ SOAPY_SDR_API unsigned *SoapySDRDevice_readRegisters(const SoapySDRDevice *devic
  */
 SOAPY_SDR_API SoapySDRArgInfo *SoapySDRDevice_getSettingInfo(const SoapySDRDevice *device, size_t *length);
 
+/*!
+ * Get information on a specific setting.
+ * \param device a pointer to a device instance
+ * \param key the setting identifier
+ * \return a list of argument info structures
```
Comment (guruofquality) [inline include/SoapySDR/Device.h:1353]: Its not returning a list

Comment (ncorgan) [reply include/SoapySDR/Device.h]: Fixed

**Comment by guruofquality** [comment]
Im just waiting for something more eventful to justify bumping the ABI. Its not a good time to deal with the resulting confusion that comes out of this.

**Pull request merged**